lib/lotus/model/configuration.rb

```diff
@@ -182,6 +277,22 @@ def _check_adapter_options!(options)
           raise ArgumentError.new("missing keyword: #{keyword}") if !options.keys.include?(keyword)
         end
       end
+
+      # Default directory that contains migration files
+      #
+      # @since 0.3.0
+      # @api private
+      def _default_migration_directory
+        "#{Dir.pwd}/db/migrations"
```

Comment:
What about `Pathname.pwd.join('db', 'migrations')`? This will make the code OS independent.